CONTRIBUTING.md

```diff
@@ -83,4 +83,50 @@ libzmq, and inherits LGPL
 * perf examples are (c) iMatix, and LGPL
 
 [tornado]: http://www.tornadoweb.org
-[paramiko]: http://www.lag.net/paramiko
\ No newline at end of file
+[paramiko]: http://www.lag.net/paramiko
+
+## Setting up development environment
+
+To start development on pyzmq, you need a working development environment. Due to the dependancies on libzmq,
+this is a bit more involved than simply checking out the code.
+
+First, install the ZeroMQ libraries on your system in a place where you can find them again. Some details on
+how to accomplish this can be found in the [PyZMQ wiki]. On a Mac with [Homebrew][] you might do this:
+
+```
+brew install zeromq
+```
+
+
+Next, install the requirements for running tests, presumably in a virtual environment. Navigate to the root
+directory of the PyZMQ source and run:
+
+```
+pip install -r test-requirements.txt
+```
+You'll also need Cython, so install that too, again presumably in a virtual environment.
+
+```
+pip install cython
+```
+
+
+Finally, run the setup command to configure the code to locate the libzmq library and then again to build
+the library bindings in place. This assumes the libzmq files were installed in /usr/local as they would be
+on a Mac running Homebrew; adjust as needed for your environment.
+
+```
+python setup.py configure --zmq=/usr/local
+python setup.py build_ext --inplace
+```
+
+
+If all went according to plan you should be able to run the unit tests without a problem. For example:
+
+```
+python -m unittest zmq.tests.test_pubsub
```

Comment:
```suggestion
pytest zmq/tests/test_pubsub.py
```

Reply:
I reverted changes to CONTRIBUTING so this doesn't apply now.